src/quo2/components/list_items/dropdown.cljs

```diff
@@ -0,0 +1,148 @@
+(ns quo2.components.list-items.dropdown
```

Comment:
I think the namespace is wrong here.

it should be `(ns quo2.components.dropdowns.dropdown`
![Uploading Screenshot 2022-09-20 at 11.19.35.png…]()

Reply:
Done